lib/new_relic/agent/instrumentation/excon.rb

```diff
@@ -65,7 +65,8 @@ def install_middleware_excon_instrumentation
   end
 
   def install_legacy_excon_instrumentation
-    ::NewRelic::Agent.logger.info 'Installing legacy Excon instrumentation'
+    Deprecator.deprecate :install_legacy_excon_instrumentation, :install_middleware_excon_instrumentation, "8.0.0"
```

Comment:
I'm not sure how this is working!  I just discovered the same on the prepend_mongo branch required the full namespace: `NewRelic::Agent::Deprecator.deprecate(...)`